bundle/Controller/Content/VisibilityInfo.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\IbexaAdminUIExtraBundle\Controller\Content;
+
+use Ibexa\Contracts\AdminUi\Controller\Controller;
+use Ibexa\Contracts\Core\Repository\ContentService;
+use Ibexa\Contracts\Core\Repository\Exceptions\BadStateException;
+use Ibexa\Contracts\Core\Repository\LocationService;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+use function implode;
+
+class VisibilityInfo extends Controller
+{
+    public function __construct(
+        private readonly LocationService $locationService,
+        private readonly TranslatorInterface $translator,
+        private readonly ContentService $contentService,
+    ) {}
+
+    public function __invoke(int $contentId, Request $request): Response
+    {
+        $iconPath = $request->attributes->get('iconPath') ?? $request->query->get('iconPath');
```

Comment:
As far as I can see, we can use `IconPathResolverInterface` to resolve `hide` icon path here, without passing it as a parameter.